src/components/dialogs/parameters/load-flow-parameters.js

```diff
@@ -253,12 +253,9 @@ const AdvancedParameterButton = ({ showOpenIcon, label, callback }) => {
     );
 };
 
-const AdvancedLoadFlowParameters = ({
-    lfParams,
-    commitLFParameter,
-    showAdvancedLfParams,
-    setShowAdvancedLfParams,
-}) => {
+const AdvancedLoadFlowParameters = ({ lfParams, commitLFParameter }) => {
+    const [showAdvancedLfParams, setShowAdvancedLfParams] = useState(false);
```

Comment:
moving these states into inner components has a little behaviour impact:
when we expand "advanced parameters" and/or "specific parameters", and then we visit others tabs from the Parameters dialog, and go back to LF parameters,  then "advanced parameters" and/or "specific parameters" are not expanded anymore.

This is a small acceptable change (to me), but I wanted to report it.

Reply:
You are right ! It's a regression.
Fixed, tell we if it's ok now ?

Reply:
good!